Neuro/Minigames/Solvers/ChartCourseSolver.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using Neuro.Cursor;
+using UnityEngine;
+
+namespace Neuro.Minigames.Solvers;
+
+[MinigameSolver(typeof(CourseMinigame))]
+public class ChartCourseSolver : MinigameSolver<CourseMinigame>
+{
+    protected override IEnumerator CompleteMinigame(CourseMinigame minigame, NormalPlayerTask task)
+    {
+        // for some reason this minigame uses localPositions for everything so we have to convert them into world positions
+
+        yield return InGameCursor.Instance.CoMoveTo(minigame.Ship);
+        InGameCursor.Instance.StartHoldingLMB(minigame.Ship);
+        for (int i = 0; i < minigame.NumPoints; i++)
```

Comment:
This minigame saves its state. If it's interrupted by a meeting for example, the ship will start at the last point that it had crossed previously. This means that the cursor will move to the first points for no reason. Consider starting this for not from 0, but from the current point that the ship is at.

![image](https://user-images.githubusercontent.com/32238504/230722683-62420475-9f24-4403-ab92-9b6e470c18ad.png)

Reply:
Good to know